examples/with-pm2/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "with-pm2",
+	"version": "1.0.0",
+	"main": "dist/index.js",
+	"author": "@sapphiredev",
+	"license": "MIT",
+	"dependencies": {
+		"@discordjs/collection": "^0.2.1",
+		"@sapphire/decorators": "next",
+		"@sapphire/discord-utilities": "^2.1.5",
+		"@sapphire/discord.js-utilities": "next",
+		"@sapphire/fetch": "next",
+		"@sapphire/framework": "next",
+		"@sapphire/plugin-api": "next",
+		"@sapphire/plugin-logger": "next",
+		"@sapphire/plugin-subcommands": "next",
+		"@sapphire/time-utilities": "next",
+		"@sapphire/type": "^2.0.11",
+		"@sapphire/utilities": "^2.0.1",
+		"@skyra/editable-commands": "^2.1.0",
+		"discord.js": "^13.0.1"
+	},
+	"devDependencies": {
+		"@sapphire/prettier-config": "^1.1.6",
+		"@sapphire/ts-config": "^3.0.0",
+		"@types/node": "^16.4.0",
+		"@types/ws": "^7.4.7",
+		"npm-run-all": "^4.1.5",
+		"pm2": "^5.1.0",
+		"prettier": "^2.3.2",
+		"typescript": "^4.3.5"
+	},
+	"scripts": {
+		"build": "tsc",
+		"watch": "tsc -w",
+		"start": "pm2 start ecosystem.config.js",
```

Comment:
Maybe add a `pm2:start` and just have the start command use that. Just for the sake of consistency